src/igvfd/audit/biosample.py

```diff
@@ -169,3 +169,27 @@ def audit_multiple_ics_with_mismatched_access(value, system):
                 f'{", ".join(links_by_dul)}.'
             )
             yield AuditFailure(audit_message_dul.get('audit_category', ''), f'{detail} {audit_message_dul.get("audit_description", "")}', level=audit_message_dul.get('audit_level', ''))
+
+
+@audit_checker('Biosample', frame='object')
+def audit_annotated_from_virtual(value, system):
+    '''
+    [
+        {
+            "audit_description": "Biosamples should not be annotated_from a virtual sample.",
```

Comment:
Field names shouldn't be included in the description for readability. (remove the underscore)